automation/src/test/scala/org/broadinstitute/dsde/workbench/NotebooksCanaryTest.scala

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.dsde.workbench
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.leonardo.notebooks.{Notebook, NotebookTestUtils, Python2, Python3}
+import org.scalatest.Matchers
+import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
+import org.broadinstitute.dsde.workbench.leonardo.Leonardo.ApiVersion.V2
+import org.broadinstitute.dsde.workbench.model.google.{GcsObjectName, GcsPath, GoogleProject}
+import org.scalatest.{FreeSpec, ParallelTestExecution}
+import org.broadinstitute.dsde.workbench.leonardo.{cluster, _}
+
+
+class NotebooksCanaryTest extends FreeSpec with Matchers with NotebookTestUtils with ParallelTestExecution with
+  BillingFixtures {
+
+
+
+  implicit val authToken: AuthToken = ronAuthToken
+  "Test for creating a cluster and localizing a notebook" - {
```

Comment:
nit: i wonder if this should be called something like "Leonardo Canary Test" and the following block, which contains the actual test, can be the description of what should happen